src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -311,7 +309,6 @@ private void RegisterLocalServices()
             this.localServices.Add(typeof(ITestProjectRegexSetter), projectNameTestProjectIndicator);
 
             this.localServices.Add(typeof(IProjectSystemHelper), new Lazy<ILocalService>(() => new ProjectSystemHelper(this, projectToLanguageMapper)));
-            this.localServices.Add(typeof(IRuleSetInspector), new Lazy<ILocalService>(() => new RuleSetInspector(this, Logger)));
```

Comment:
Simpler fix up this time; the code that used this interface had already been deleted, so this was pretty much a straightforward delete of the unused code.

I couldn't find any specific resources strings used by the deleted code.